Makefile

```diff
@@ -175,15 +159,24 @@ endif
 # pairs are stored in the `compilers_engines` variable.
 compilers_engines :=
 
-ifeq ($(IS_AMD64), 1)
-	compilers_engines += cranelift-jit
+##
+# The Cranelift case.
+##
 
-	ifeq ($(IS_WINDOWS), 0)
-		# `cranelift-native`
-		ifneq ($(LIBC), musl)
-			# Native engine doesn't work on Windows and musl yet.
+compilers_engines += cranelift-jit
+
+ifeq ($(IS_WINDOWS), 0)
```

Comment:
MHO we should stop testing for "not OS <X>" and instead test for "is OS <X>".

Reply:
Yup. I was hesitant. `Makefile` has no logical OR operator. We need to use a trick like:

```
ifneq (, $(filter 1, $(IS_DARWIN) $(IS_LINUX)))
```

When you know it, that's OK, but at first glance, it feels weird.

Reply:
(code updated)